main.js

```diff
@@ -18,10 +17,17 @@ var formSearch = document.querySelector("#search-form");
 saveButton.addEventListener("click", saveIdea);
 
 //functions:
-// we need a save function
-    // needs to be able to update the card title and body input field
-function saveIdea(){
-    var newCard = new Ideas(formTitle.value, formBody.value) {
+function saveIdea(event) {
```

Comment:
This section looks really good, It's a little big I wonder if we can make it smaller